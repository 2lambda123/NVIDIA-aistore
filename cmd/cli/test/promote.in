mkdir -p /tmp/prm && echo 012345 > /tmp/prm/obj
ais create bucket $BUCKET // IGNORE
ais promote /tmp/prm/obj ais://$BUCKET/one/one // IGNORE
ais promote /tmp/prm/obj ais://$BUCKET/two/ // IGNORE
ais promote /tmp/prm/ ais://$BUCKET/three // IGNORE
ais promote /tmp/prm ais://$BUCKET/fourth // IGNORE
ais promote /tmp/prm ais://$BUCKET/fifth/ // IGNORE
ais promote /tmp/prm/ ais://$BUCKET/sixth/ // IGNORE

ais ls ais://$BUCKET

rm -rf /tmp/prm // IGNORE
ais rm bucket $BUCKET // IGNORE
