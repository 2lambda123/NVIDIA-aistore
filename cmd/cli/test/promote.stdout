^.*promoted.*"/tmp/prm/obj".*=>.*$
^.*promoted.*"/tmp/prm/discard".*=>.*$
^.*promoted.*"/tmp/prm/".*=>.*$
^.*promoted.*"/tmp/prm".*=>.*$
^.*promoted.*"/tmp/prm".*=>.*$
^.*promoted.*"/tmp/prm/".*=>.*$
NAME             SIZE
fifth/obj        7B
fourth/obj       7B
one/one          7B
sixth/obj        7B
three/obj        7B
two/discard      7B
